### PR TITLE
Fixes `Terminal: Focus Terminal Tabs view` command not working

### LINKED ISSUE #163789: Terminal: Focus Terminal Tabs view keybinding appears dead/non-working.
I'm on macOS 12.6 running VS Code version 1.72.2 (Universal) and after a recent update, the "Terminal: Focus Terminal Tabs View" keybinding won't function.

In short, while focused on the terminal pane, I used to be able to press cmd+shift+\ and move the focus to the Terminal Tabs to the right of the terminal pane (if I had more than one terminal open, which caused the Terminal Tabs to exist).

Now, that keybinding doesn't work anymore, nor does any alternative keybinding for "Terminal: Focus Terminal Tabs". 

These are the current when expressions:
"terminalFocus && terminalHasBeenCreated || terminalFocus && terminalProcessSupported || terminalHasBeenCreated && terminalTabsFocus || terminalProcessSupported && terminalTabsFocus"

But removing those when expression entirely or altering them or testing them one at a time does nothing to solve this issue either.

### PR DESCRIPTION
**Pull request by jeanp413**
Fixes #163789

Regression from https://github.com/microsoft/vscode/pull/161989

### CODE REVIEW

File: src/vs/workbench/contrib/terminal/browser/terminalView.ts [hunk 1]
```diff
@@ -47,7 +47,6 @@ import { ServicesAccessor } from 'vs/editor/browser/editorExtensions';
 import { TerminalCapability } from 'vs/platform/terminal/common/capabilities/capabilities';
 
 export class TerminalViewPane extends ViewPane {
-	private _actions: IAction[] | undefined;
```
Comment (jeanp413) [inline src/vs/workbench/contrib/terminal/browser/terminalView.ts:50]: minor cleanup :broom:

File: src/vs/workbench/contrib/terminal/browser/terminalView.ts [hunk 2]
```diff
@@ -164,11 +158,11 @@ export class TerminalViewPane extends ViewPane {
...
 			if (visible) {
-				const hadTerminals = !!this._terminalGroupService.groups.length;
-				// Ensure the primary backend is registered as it's important to do before
-				// initializeTerminals is called.
-				await this._terminalService.primaryBackendRegistered;
 				if (this._terminalService.isProcessSupportRegistered) {
+					const hadTerminals = !!this._terminalGroupService.groups.length;
+					// Ensure the primary backend is registered as it's important to do before
+					// initializeTerminals is called.
+					await this._terminalService.primaryBackendRegistered;
```
Comment (jeanp413) [inline src/vs/workbench/contrib/terminal/browser/terminalView.ts:165]: this was awaiting forever in serverless vscode as there's no backend

File: src/vs/workbench/contrib/terminal/browser/terminalGroupService.ts [hunk 3]
```diff
@@ -181,12 +181,7 @@ export class TerminalGroupService extends Disposable implements ITerminalGroupSe
 				}
 				// Do not await the focus as setVisible must be called immediately to ensure
 				// something else didn't steal focus
-				instance.focusWhenReady(true);
-				// HACK: as a workaround for https://github.com/microsoft/vscode/issues/134692,
-				// this will trigger a forced refresh of the viewport to sync the viewport and scroll bar.
-				// This can likely be removed after https://github.com/xtermjs/xterm.js/issues/291 is
-				// fixed upstream.
-				instance.setVisible(true);
+				await instance.focusWhenReady(true);
```
Comment (jeanp413) [inline src/vs/workbench/contrib/terminal/browser/terminalGroupService.ts:184]: I think it's save to remove this hack after the refactor in https://github.com/microsoft/vscode/pull/149621, I can't reproduce it anymore. Also tested https://github.com/microsoft/vscode/issues/160914 and is working fine too

**Comment by jeanp413** [comment]
cc @meganrogge @Tyriar

**Review by meganrogge — APPROVED** [review]
Thanks 👍🏼

**Pull request merged**